index.html

```diff
@@ -14,8 +14,34 @@
   </head>
   <body>
     <section id="content-1">
+      <div id="ham-menu">
+        <!-- Becky Add X Image -->
+        <img
+          onclick="clickX()"
+          id="x"
+          src="https://img.icons8.com/material-outlined/24/null/delete-sign.png"
+        />
+        <ul>
+          <li>
+            <a onclick="clickX()" href="#portfolio" class="clr-grey"
+              >Portfolio</a
+            >
+          </li>
+          <li>
```

Comment:
Kindly ensure that the design of the menu button is as close to the design as possible and looks same on mobile sizes.

Your design:
![image](https://user-images.githubusercontent.com/20554583/231131261-99d5a766-651a-4384-9840-eac1404c68f5.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/231131410-82905913-9422-4c01-a1a0-92dddbdc0226.png)

Notice how the dropdown covers the width of the screen.